requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+flake8
```

Comment:
Got it, thx!  My previous project was moving towards setuptools, so I guess it was headed in the wrong direction 😱